storm-core/pom.xml

```diff
@@ -464,6 +464,30 @@
                 </dependencies>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```

Comment:
This code will not run on a build for Windows.  I think it is going to produce build errors if we try.  I would like to see this placed in a profile that is activated only for Operating systems that we know it works on. And file a follow up JIRA to make this work on Windows.

https://maven.apache.org/guides/introduction/introduction-to-profiles.html